src/components/map.jsx

```diff
@@ -23,21 +23,52 @@ let mapProp = {
 // this.ref is referencing the DOM element(used instead of getElement)
 this.mapObj = new google.maps.Map(this.refs.myMap, mapProp);
 
+let submit = document.getElementById('submit');
```

Comment:
What we can do in the next PR, is update this section to utilize the onClick event on the button itself! For now this is fine